wandb/sdk/launch/launch_add.py

```diff
@@ -152,6 +160,23 @@ def _launch_add(
         cuda,
         run_id,
     )
+
+    if build:
+        if launch_spec.get("job") is not None:
+            wandb.termwarn("Build doesn't support setting a job. Overwriting job.")
+            launch_spec["job"] = None
+
+        launch_project = create_project_from_spec(launch_spec, api)
+        docker_image_uri = build_image_from_project(
+            launch_project, api, build_type="docker"
```

Comment:
Lets try to source the `build_type` from the `LAUNCH_CONFIG_FILE` and default to docker. This will make supporting other builders smoother in the future